packages/block-editor/src/components/use-block-sync/block-id-map.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * External dependencies
+ */
+const { v4: uuid } = require( 'uuid' );
```

Comment:
Why is this using `require` instead of `import`?

Reply:
copy/pasted from the wrong place! good catch

Reply:
cb2ac71